### ISSUE TITLE
Question about Loading Page

### ISSUE THREAD

**Issue by nungnung** [swsnu/swppfall2019#94]
로딩 페이지를 넣어서 데이터를 불러오는 와중에는 로딩 페이지가 떴다가 완료되면 나타나야 할 페이지가 뜨도록 하였는데 괜찮은지 궁금합니다.
Could I use Loading Page?

**Comment by ktaebum** [comment]
Yes (actually it's good to implement `Loading` when considering real usage). In fact,  we can implement loading without splitting page, but it's okay
